.travis.yml

```diff
@@ -31,9 +31,12 @@ script:
  - flake8 --select BLK *.py
  - echo "Checking we report no errors on these test cases"
  - flake8 --select BLK tests/test_cases/*.py
+ - flake8 --select BLK tests/with_pyproject_toml/*.py
+ - flake8 --select BLK test/non_conflicting_configurations/*.py
```

Comment:
test -> tests

Reply:
Ah, the files ought to be moved too - we only need one top level ``tests/`` folder ;)

Reply:
I double checked - all files are on the right place) Typo in travis was fixed - please check